Cross.toml

```diff
@@ -21,3 +21,6 @@ image = "criticalstack/cross:x86_64-apple-darwin"
 
 [target.x86_64-unknown-linux-gnu]
 image = "criticalstack/cross:x86_64-unknown-linux-gnu"
+
+[target.aarch64-apple-darwin]
+image = "criticalstack/cross:aarch64-apple-darwin"
```

Comment:
We're going to want to update the `aarch64-apple-darwin` image to something that actually exists, and we're going to have to verify it works before we merge this in. Currently, the `criticalstack/cross:aarch64-apple-darwin` image doesn't exist and we need to find out if we can actually create that image